content/chainguard/vms/compliance-features.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Chainguard VMs Compliance Features"
+linktitle: "Compliance"
+description: "Learn about supported compliance features for Chainguard VMs"
+type: "article"
+date: 2025-11-20T08:04:00+00:00
+lastmod: 2025-11-20T15:09:59+00:00
+draft: false
+tags: ["Chainguard VMs", "Compliance"]
+menu:
+  docs:
+    parent: "vms"
+weight: 005
+toc: true
+---
+
+Chainguard VMs provide pre-hardened, audit-ready Linux virtual machine images designed for regulated and high-assurance environments (federal, defense, healthcare, financial services, and suppliers to those sectors). These images combine the following features:
+
+| Feature                             | Description |
+|:------------------------------------| :---- |
+| **FIPS 140-3 validated crypto**     | [NIST](https://www.nist.gov/) CMVP-validated software modules and [SP 800-90B](https://csrc.nist.gov/pubs/sp/800/90/b/final) compliant entropy, with runtime guardrails blocking non-FIPS crypto. |
+| **STIG hardening**                  | Pre-configured to DISA [STIG](https://edu.chainguard.dev/chainguard/chainguard-images/features/image-stigs/) controls; delivered as production-ready images. |
+| **CIS benchmark compliance**        | [CIS](https://www.cisecurity.org/cis-benchmarks/cis-benchmarks-faq) Level 1 hardened variants; hybrid STIG \+ CIS baseline. |
+| **Secure Boot**                     | Secure Boot enabled by default across AWS, Azure, GCP, and on-prem. |
+| **Compliance evidence & reporting** | FIPS certificates, OpenSSL docs, SCAP scan results, and POA\&M-ready artifacts. |
```

Comment:
```suggestion
| **Compliance evidence & reporting** | FIPS certificates, OpenSSL docs, Security Content Automation Protocol (SCAP) scan results, and POA\&M-ready artifacts. |
```

This comes up a couple times in this doc and I wasn't sure what it meant off the top of my head